packages/ag-charts-community/src/chart/data/processors.ts

```diff
@@ -112,7 +112,7 @@ export function trailingAccumulatedValueProperty<K>(
 export function groupAccumulativeValueProperty<K>(
     propName: K,
     mode: 'normal' | 'trailing' | 'window' | 'window-trailing',
-    sum: 'current' | 'last' = 'current',
+    sum: 'current' | 'last',
```

Comment:
This is never used without specifying this parameter and the next param `opts` is required.